ui/desktop/electron-app/config/desktop.js

```diff
@@ -0,0 +1,63 @@
+const fs = require('fs');
+const path = require('path');
+
+// Create config for use in electron forge
+const createConfig = () => {
+  const configPath = path.join(__dirname, '..', 'config');
+  const macosConfigPath = path.join(configPath, 'macos');
+  const appIconPath = path.join(configPath, 'app-icons', 'icon');
+
+  const config = {
+    name: 'boundary',
+    productName: 'Boundary',
+    bundleId: 'com.electron.boundary',
+    copyright: `Copyright © ${new Date().getFullYear()} HashiCorp, Inc.`,
+    releaseVersion: process.env.RELEASE_VERSION,
+    releaseCommit: process.env.RELEASE_COMMIT,
+    icon: appIconPath,
+    macos: {
+      hardenedRuntime: true,
+      entitlements: path.join(macosConfigPath, 'entitlements.plist'),
+      signatureFlags: 'library',
+      signingIdentity: process.env.BOUNDARY_DESKTOP_SIGNING_IDENTITY,
+      dmg: {
+        diskIcon: path.join(macosConfigPath, 'disk.icns'),
+        background: path.join(macosConfigPath, 'background.png'),
+      },
+    },
+  };
+
+  if (!config.releaseVersion) {
+    config.releaseVersion = '0.0.0';
+    console.warn(
+      `[desktop-config] Release version not specified. Using: ${config.releaseVersion} instead`
```

Comment:
This is a tough read since the variable being used "instead" is the one that was missing in the first place.

Reply:
Instead of raising a warning, updated desktop config to set a release version if missing.